docs/otoc/otoc_example.ipynb

```diff
@@ -0,0 +1,761 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "id": "Wwf0-n0kQEFr"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Ancilla qubit: (5, 1)\n",
+      "Measurement qubit: (4, 1)\n",
+      "Butterfly qubit: (5, 2)\n",
+      "\n",
+      "Example OTOC circuit:\n",
+      "\n",
+      "(4, 1): ───Y^0.5────────@───X^-0.5───────────@───PhX(0.75)^0.5────────────────PhX(-0.25)^0.5───@───X^0.5────────────@─────────────────────────\n",
+      "                        │                    │                                                 │                    │\n",
+      "(4, 2): ───Y^0.5────────┼───PhX(0.75)^0.5────@───PhX(-0.25)^0.5───@───────@───PhX(0.75)^0.5────@───PhX(-0.25)^0.5───┼─────────────────────────\n",
+      "                        │                                         │       │                                         │\n",
+      "(5, 1): ───PhX(0)^0.5───@────────────────────────Y────────────────┼───────┼───Y─────────────────────────────────────@───PhX(0)^0.5───M('z')───\n",
+      "                                                                  │       │\n",
+      "(5, 2): ───Y^0.5────────────Y^0.5────────────@───X^-0.5───────────@───Y───@───X^0.5────────────@───Y^-0.5─────────────────────────────────────\n",
+      "                                             │                                                 │\n",
+      "(5, 3): ───Y^0.5────────────PhX(-0.25)^0.5───@───X^-0.5───────────@───────@───X^0.5────────────@───PhX(0.75)^0.5──────────────────────────────\n",
+      "                                                                  │       │\n",
+      "(6, 3): ───Y^0.5────────────X^-0.5───────────@───PhX(-0.75)^0.5───@───────@───PhX(0.25)^0.5────@───X^0.5──────────────────────────────────────\n",
+      "                                             │                                                 │\n",
+      "(7, 3): ───Y^0.5────────────PhX(0.75)^0.5────@───X^0.5────────────@───────@───X^-0.5───────────@───PhX(-0.25)^0.5─────────────────────────────\n",
+      "                                                                  │       │\n",
+      "(7, 4): ───Y^0.5────────────PhX(0.25)^0.5────@───PhX(0.25)^0.5────@───────@───PhX(-0.75)^0.5───@───PhX(-0.75)^0.5─────────────────────────────\n",
+      "                                             │                                                 │\n",
+      "(7, 5): ───Y^0.5────────────PhX(-0.25)^0.5───@───PhX(-0.25)^0.5───────────────PhX(0.75)^0.5────@───PhX(0.75)^0.5──────────────────────────────\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Ancilla qubit:\", ancilla)\n",
+    "print(\"Measurement qubit:\", qubits[0])\n",
+    "print(\"Butterfly qubit:\", q_b)\n",
+    "\n",
+    "print(\"\\nExample OTOC circuit:\\n\", otoc_circuits.butterfly_Y[0], sep=\"\\n\")"
```

Comment:
on my jupyter, whenever one does `print(circuit)` it wraps horribly and you can't tell what's going on but if you just have the last line of the cell be `circuit` it uses a horizontal scrollbar and I can actually read it

Reply:
Changed to just `circuit` instead of `print(circuit)`